nucypher/characters/lawful.py

```diff
@@ -615,7 +615,7 @@ def decryptor(ciphertext):
 
         return encrypted_treasure_map.decrypt(decryptor)
 
-    def retrieve_cfrags(
+    def try_retrieve(
```

Comment:
This function name is pretty vague. 

Perhaps something more explicit like `retrieve_without_decrypt`? or `retrieve_cfrags_for_kits` or `get_cfrags_for_kits`...?

Reply:
Renamed to `retrieve()`/`retrieve_and_decrypt()`.